### ISSUE TITLE
HHC in HTMLHelp has no encoding set

### ISSUE THREAD

**Issue by jelovirt** [dita-ot/dita-ot#1151]
Converted from [SourceForge issue 3445877](http://sourceforge.net/support/tracker.php?aid=3445877), submitted by [jelovirt](/jelovirt)

The HHC files generated by HTMLHelp plug-in do not have the encoding defined with:

``` xml
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

before convert-lang is run. This results in wrong encoding to be used in table of contents.

The HHC file is generated with XML output mode, whereas the actual topic files are with HTML output. This will make the XSLT processor to generate the meta element and allow convert-lang to correctly identify the encoding and change it.

**Comment by robander** [comment]
Thinking this may be a duplicate of #1271, which was opened a bit later but has more discussion.

**Comment by robander** [comment]
Fixed with #2852
